addons/enhancedVideo/CfgVideoOptions.hpp

```diff
@@ -0,0 +1,180 @@
+class CfgVideoOptions
+{
+	class Visibility
+	{
+		minValue=5;
+		maxValue=40000;
+	};
+	class ObjectsVisibility
+	{
+		minValue=5;
+		maxValue=30000;
+	};
+	class ShadowsVisibility
+	{
+		minValue=5;
+		maxValue=3000;
+	};
+	class Particles
+	{
+		class High
```

Comment:
You probably want to rename that class to "Ultra" or something, as "High" with `text = "High";` is already defined